### DISCUSSION TITLE
Question about choosing simulation model and where to find pKa values

### DISCUSSION THREAD

**Discussion by Gre-ux** [Open-Systems-Pharmacology/Forum#1580] [Questions & Problems]
Hi! I am looking for examples of PBPK models of peptide drug, and I found this inulin model in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5845054/
Inulin was modeled as a small molecule in Large Molecule Model referring to the material provided on this website. Inulin (with no Fc part) was assumed not metabolized and all excreted in urine. Yet in PK-SIM, the reaction diagram is like
<img width="403" alt="1690616857883" src="https://github.com/Open-Systems-Pharmacology/Forum/assets/114388923/b4f50625-63ae-4774-9ab4-6d177900ee78">
I am confused:
1. Is this a mistake and what can I do to change the reaction diagram to comply with the true process in body?
2. I am trying to build a model of a peptide drug, which is metabolized by peptidases and excreted with form of metabolites. The molecule weight is similar to inulin. If I model it as a small molecule in small molecule model, is pKa a necessary parameter? I searched a lot of websites( drugbank, pubchem, etc.)  but cannot find pKa of this drug. Can you give me some advice on how to find this value?
Looking forward for your help!

**Comment by Christoph27** [comment]
Hi!

1. This is no mistake. The reaction is always present structurally if you use the large molecules model. For the inulin model, the compound type is set to "small molecule" so that it is not getting to the endosome. So, the FcRn binding becomes irrelevant although the reaction is present. So, nothing to modify for inulin. 
2. The pKa value is used for a) calculating solubility in the different segments of the gastrointestinal tract, b) for calculation of partition coefficients only if you use the methods by Rodgers/Rowland or Schmitt and c) for calculation the permeability across cell membranes (interface interstitial - cell) in case of the charge-dependent methods of Schmitt only. 
a) should be irrelevant in your case.
Regarding b) and c): I wouldn't trust the calculation methods for this type of compounds too much anyway (& probably also your lipophilicity is not extremely reliable?). So, I would say the pKa value is not so relevant here and you could go without it.   
I would consider in your case to define your peptide as "small molecule" (molecule can diffuse into cells to some extend) but to use the "large molecule" model, so that you have the barrier between the plasma and interstitial space from the "large molecule" model (as it was done for inulin). Then you have no clearance in the endosome but the clearance from your peptidases & the renal clearance should be much larger anyway.

**Reply by Gre-ux** [reply]
Thanks very much for your reply! It really helps me understand better about these parameters in modelling!
